recipes/gklib/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set name = "gklib" %}
+{% set version = "0.0.1.dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Waiting for https://github.com/KarypisLab/GKlib/issues/5
```

Comment:
Would that be ok to merge as it is? I will release a new version once a git tag is added. I need this lib to build another package (dgl).

Reply:
It is probably best to wait

Reply:
Version 5.1.1 has been tagged.